plugins/tiddlywiki/browser-storage/util.js

```diff
@@ -0,0 +1,106 @@
+/*\
+title: $:/plugins/tiddlywiki/browser-storage/util.js
+type: application/javascript
+module-type: library
+
+Utility methods for browser-storage plugin
+
+\*/
+
+(function(){
+
+    /*jslint node: true, browser: true */
```

Comment:
There's one extra level of indentation than we would normally use for a JS module.

Reply:
I removed the indentation.

Reply:
I will likely look into the additional change to enable coordinating with the syncer logic, but in my opinion the current pull request meets my original intent. Would it be better to create a separate pull request for the additional change or create a branch from this pull request and change the base branch later, if and when it is suitable for review?

Reply:
Thanks @fkmiec, I think a separate PR would be easier to manage.